ConsoleApp1/Program.cs

```diff
@@ -1,7 +1,22 @@
-﻿namespace ConsoleApp1 {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿using System;
+namespace DigitalRootFinder {
+   class Program {
+      static void Main (string[] args) {
+         Console.WriteLine ("Enter the number:");
+         int number = int.Parse (Console.ReadLine ());
+         int DigitalRoot = CalculateDigitalRoot (number);
+         Console.WriteLine ($"The digital root of {number} is {DigitalRoot}");
+      }
+      static int CalculateDigitalRoot (int num) {

```

Comment:
There is a simpler way to calculate digital root of a number in base 10 given in the wikipedia article

Reply:
Will check and change